src/bindings/html/view.js

```diff
@@ -41,13 +43,21 @@ export class View {
   formatEntity(id, args) {
     return this.ctx.formatEntity(this.service.languages, id, args);
   }
+
+  translateFragment(frag) {
+    return translateFragment(this, frag);
+  }
 }
 
 View.prototype.setAttributes = setL10nAttributes;
 View.prototype.getAttributes = getL10nAttributes;
-View.prototype.translateFragment = translateFragment;
 
-function onMutations(mutations) {
+export function init(langs) {
+  dispatchEvent(this.doc, 'supportedlanguageschange', langs);
+  return translateDocument(this, this.doc, langs);
```

Comment:
Yeah, I'm not sure where `supportedlanguageschange` belongs.  If not here, than we don't need `View#init` at all, since all else it does is `translateDocument`.